packages/core/src/renderer/rehydrateRendererCache.test.ts

```diff
@@ -9,7 +9,6 @@ describe('rehydrateRendererCache', () => {
     styleElement.setAttribute(DATA_BUCKET_ATTR, 'd');
     document.head.appendChild(styleElement);
     styleElement.textContent = '.foo { color: red; }';
-
     rehydrateRendererCache(renderer, document);
```

Comment:
Merge conflict?

Reply:
yeah, I tried to squash commits manually and it was an awful experience